painttiming.bs

```diff
@@ -62,9 +83,25 @@ Formally, we consider the user agent to have "rendered" a document when it has p
 
     NOTE: The rendering pipeline is very complex, and the timestamp should be the latest timestamp the user agent is able to note in this pipeline (best effort). Typically the time at which the frame is submitted to the OS for display is recommended for this API.
 
+<dfn>Contentful</dfn>: An element counts as contentful when one or more of the following apply:
+* The element has a [=text node=] descendant, with at least one character, excluding [=document white space characters=].
```

Comment:
This doesn't seem right. Assuming this refers to the [concept of tree descendant](https://dom.spec.whatwg.org/#concept-tree-descendant), this means that every element which has a character is assumed to be contentful even if that text is inside another element with `display: none`. That doesn't seem right.

Reply:
Will revise.